src/test/java/org/skife/jdbi/v2/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -67,9 +72,10 @@ public Object withHandle(Handle handle) throws Exception
         public String findNameById(@Bind long id);
     }
 
-    @Ignore
     @Test
     public void testFoo() throws Exception {
+        assumeTrue(postgresIsInstalled);
```

Comment:
I believe, unfortunately, JUnit doesn't handle assumptions in the `@Before` or `@After` auxiliary methods. 

See [this](http://stackoverflow.com/questions/30112317/how-does-assumetrue-work-in-junit) (unfairly downvoted) SO question for the clarification.